ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:14-alpine3.11
```

Comment:
adonisjs has dependency "Knex" this package don't work with nodejs version 14 please use node 12 version